src/webgl/p5.RendererGL.Immediate.js

```diff
@@ -371,39 +427,23 @@ p5.RendererGL.prototype._tesselateShape = function() {
 p5.RendererGL.prototype._drawImmediateFill = function() {
   const gl = this.GL;
   this._useVertexColor = (this.immediateMode.geometry.vertexColors.length > 0);
-  const shader = this._getImmediateFillShader();
+
+  let shader;
+  shader = this._getImmediateFillShader();
 
   this._setFillUniforms(shader);
 
   for (const buff of this.immediateMode.buffers.fill) {
     buff._prepareBuffer(this.immediateMode.geometry, shader);
   }
 
-  // LINE_STRIP and LINES are not used for rendering, instead
```

Comment:
Why does this PR negate the need for this block of code?

Reply:
I moved it up to around line 198. The idea is that we always need to do this step whether we're actually rendering to the screen or whether we're putting it into a p5.Geometry, so I factored it out of this function (which just does rendering) and put it in a spot where both paths run it.